wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,74 @@
+## What is considered cheating?
+The rules say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule. Cheaters will be banned on their first offence and for an indefinite amount of time. The team may decide to lift their ban if they successfully appeal.
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
-osu! uses an automated metrics combined with manual confirmation.
+## I have been banned for cheating. I didn't cheat!
+osu! uses automated metrics combined with manual confirmation.
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
+The team never places an incorrect ban for cheating. Before they take any action, they make sure it is 100% that you cheated.
 
-My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
---------------------------------------------------------------------------------------
 
-It may be hard to come to terms with, but you will have to trust the staff. peppy is not trying to tear apart friendships but he knows that nearly no one can be trusted. Even those people who are close to you that you may never suspect can be just acting behind you or any other person.
+## My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
+We understand that you trust your friends, but you will have to believe the team. Punishments aren't issued unless there is sufficient evidence to support the decision. Whether or not they have cheated, it shouldn't affect your friendship!
 
-Prove that I/they were cheating! Tell us how you know!
-------------------------------------------------------
 
-The staff cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
+## Prove that I/they were cheating! Tell us how you know!
+The team cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
 
-I'll make a video to prove my innocence!
-----------------------------------------
 
+## I'll make a video to prove my innocence!
 Sorry but if you have been banned, they are already 100% sure you are cheating. A video will not help.
 
-I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
------------------------------------------------------------------------------------------------
 
-Stay calm.
+## I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
+Stay calm and **don't discuss their play in public chat**.
+Don't give them undue attention or ask others to spectate them. There are three ways to report users.
```

Comment:
There should be a form of transition between those two sentences, otherwise they feel too disconnected. Add "You should report the user and not mention them anywhere." and then change "report users" (last sentence) to "do so".